app/src/main/res/values/strings.xml

```diff
@@ -835,6 +835,7 @@ Otherwise, you can download another keyboard in the app store. Popular keyboards
     <string name="team_mode">Team Mode</string>
     <string name="team_mode_exit">Exit Team Mode</string>
     <string name="team_mode_description">When mapping the same area together in a group, you can enter team mode to split up quests between all team members.</string>
+    <string name="team_mode_description_additional_tutorial">Every team member needs to enable team mode on their device. Remember to disable it after shared mapping!</string>
```

Comment:
I'm no native speaker, but "shared mapping" sound wrong. @peternewman Can you help here?

```suggestion
    <string name="team_mode_description_additional_tutorial">Every team member needs to enable team mode on their device. Remember to disable it after mapping together!</string>
```

Reply:
Yeah @FloEdelmann 's "mapping together" feels better to me and echos some of the wording above, without the potential for confusion of the social media usage of "share via...".

It's probably neater than stuff I could come up with too like "...after simultaneous mapping!" or collaborative.

Reply:
@matkoniecz this seems to have either been reverted or missed at some point, as it still says shared mapping.